newIDE/app/src/UI/Theme/Global/EventsSheet.css

```diff
@@ -36,202 +36,210 @@
-    background-color: var(--event-sheet-instruction-parameter-error-background-color);
-    border-bottom: 1px dashed  var(--event-sheet-instruction-parameter-error-color);
-    min-width: 25px;
-    min-height: 14px;
+  display: inline-flex;
+  background-color: var(--event-sheet-instruction-parameter-error-background-color);
+  border-bottom: 1px dashed  var(--event-sheet-instruction-parameter-error-color);
+  min-width: 25px;
+  min-height: 14px;
 }
 
 /**
  * Search results highlight
  */
 .gd-events-sheet .with-search-results .rst__rowSearchMatch {
-    outline: none;
+  outline: none;
 }
-.gd-events-sheet .with-search-results .rst__rowSearchMatch .rst__moveHandle {
-    background: var(--event-sheet-events-highlighted-color);
+.gd-events-sheet .with-search-results .rst__rowSearchMatch .moveHandle {
+  background: var(--event-sheet-events-highlighted-color);
 }
-.gd-events-sheet .with-search-results .rst__rowSearchMatch .rst__moveHandle:hover {
-    filter: brightness(120%);
+.gd-events-sheet .with-search-results .rst__rowSearchMatch .moveHandle:hover {
+  filter: brightness(120%);
 }
 .gd-events-sheet .with-search-results .rst__rowSearchMatch .conditions-container {
-    outline: none;
-    background-color: var(--event-sheet-events-highlighted-background-color);
+  outline: none;
+  background-color: var(--event-sheet-events-highlighted-background-color);
 }
 .gd-events-sheet .with-search-results .rst__rowSearchMatch .actions-container {
-    outline: none;
-    background-color: var(--event-sheet-events-highlighted-background-color);
+  outline: none;
+  background-color: var(--event-sheet-events-highlighted-background-color);
 }
 
-.gd-events-sheet .with-search-results .rst__rowSearchFocus .rst__moveHandle {
-    width: 15px;
+.gd-events-sheet .with-search-results .rst__rowSearchFocus .moveHandle {
+  width: 15px;
 }
 
 /**
-* Drag'n'drop indicator (events)
+* Drag'n'drop indicator (events - instructions)
 */
 .gd-events-sheet .drop-indicator {
-    border-top: 2px solid var(--event-sheet-drop-indicator-can-drop-border-top-color);
-    height: 0;
-    margin-top: -1px;
-    margin-bottom: -1px;
-    width: 100%;
-    box-sizing: border-box;
+  border-top: 2px solid var(--event-sheet-drop-indicator-can-drop-border-top-color);
+  height: 0;
+  margin-top: -1px;
+  margin-bottom: -1px;
+  width: 100%;
+  box-sizing: border-box;
 }
 
 .gd-events-sheet .cant-drop-indicator {
-    border-top: 2px solid var(--event-sheet-drop-indicator-can-drop-border-top-color);
-    height: 0;
-    margin-top: -1px;
-    margin-bottom: -1px;
-    width: 100%;
-    box-sizing: border-box;
+  border-top: 2px solid var(--event-sheet-drop-indicator-can-drop-border-top-color);
+  height: 0;
+  margin-top: -1px;
+  margin-bottom: -1px;
+  width: 100%;
+  box-sizing: border-box;
+}
+
+/**
+* Drag'n'drop indicator (events)
+*/
+.gd-events-sheet .event-drop-indicator {
+  border: 3px solid var(--event-sheet-event-drop-indicator-border-color);
```

Comment:
Please make this border property fully customizable in the theme 🙏

Reply:
For consistency we should have the same drop indicator (color and size) everywhere (like in the objects list, group list, etc...)